tests/mgbench/Dockerfile.mgbench_client

```diff
@@ -32,11 +42,13 @@ RUN source /opt/toolchain-v4/activate && \
     make .
 
 
-FROM ubuntu:22.04
+FROM ubuntu:24.04
 
-RUN apt-get update && apt-get install -y wget libcurl4
+RUN apt-get update && apt-get install -y wget libcurl4 python3 && \
+    rm -rf /var/lib/apt/lists/*
 
 # Copy mgbench client to clean image
-COPY --from=mg_bench_client_build_base /memgraph/build/tests/mgbench/client /bin/
+COPY --from=mg_bench_client_build_base /memgraph/build/tests/mgbench /bin
+COPY --from=mg_bench_client_build_base /memgraph/tests/mgbench /mgbench
 
-ENTRYPOINT ["bin/client"]
+ENTRYPOINT ["python3", "/mgbench/benchmark.py"]
```

Comment:
Yeah, this `benchmark.py`  serves another purpose to run the actual client that is being packed in here, I think we should hop on a short call to get to the bottom of this thing.